src/gcp/cloudfunctions.ts

```diff
@@ -356,11 +356,16 @@ export async function updateFunction(
   cloudFunction: Omit<CloudFunction, OutputOnlyFields>
 ): Promise<Operation> {
   const endpoint = `/${cloudFunction.name}`;
+  // TODO(colerogers): remove cloudFunctionWithoutHttpsTrigger and set securityLevel in functionFromEndpoint during a breaking change release
+  const cloudFunctionWithoutHttpsTrigger = { ...cloudFunction };
+  delete cloudFunctionWithoutHttpsTrigger.httpsTrigger;
```

Comment:
nit* can you add a comment on why we are clearing this field (e.g. link to the issue)

Reply:
Changed the implementation to set `securityLevel` in `backend.Endpoint`. I removed this delete field code